### PR TITLE
[array_of_tuples] Fix bridges

### PR DESCRIPTION
**Pull request by blegat**

### CODE REVIEW

File: src/Bridges/rsocbridge.jl [hunk 1]
```diff
@@ -28,8 +28,8 @@ function RSOCBridge{T}(model, f::MOI.VectorAffineFunction{T}, s::MOI.RotatedSeco
     u = MOIU.eachscalar(f)[2]
     x = MOIU.eachscalar(f)[3:d]
     s2 = √2
-    y = MOI.ScalarAffineFunction([t.variables; u.variables], [t.coefficients/s2; -u.coefficients/s2], t.constant/s2 - u.constant/s2)
-    z = MOI.ScalarAffineFunction([t.variables; u.variables], [t.coefficients/s2;  u.coefficients/s2], t.constant/s2 + u.constant/s2)
+    y = MOI.ScalarAffineFunction([mapcoefficient.(c -> c/s2, t.terms); mapcoefficient.(c -> -c/s2, u.terms)], t.constant/s2 - u.constant/s2)
```
Comment (mlubin) [inline src/Bridges/rsocbridge.jl]: `mapcoefficient.(c -> c/s2, [t.terms; u.terms])`?
Same below

File: src/Bridges/soctopsdbridge.jl [hunk 2]
```diff
@@ -11,26 +6,26 @@ Builds a VectorAffineFunction representing the upper (or lower) triangular part
 [ f[2:end] g * I     ]
 """
 function _SOCtoPSDCaff(f::MOI.VectorAffineFunction{T}, g::MOI.ScalarAffineFunction{T}) where T
-    dim = length(f.constant)
+    dim = MOIU.moilength(f)
```
Comment (mlubin) [inline src/Bridges/soctopsdbridge.jl]: Uh, what is `moilength`? `output_length` maybe?

Comment (blegat) [reply src/Bridges/soctopsdbridge.jl]: I was also looking for a better name while doing this PR :-P
I have created an issue https://github.com/JuliaOpt/MathOptInterface.jl/issues/366

File: src/Bridges/rsocbridge.jl (hunk 1 above)
Comment (blegat) [reply src/Bridges/rsocbridge.jl]: The maps is different, the first map is `c/s2` and the second map is `-c/s2`.

Comment (mlubin) [reply src/Bridges/rsocbridge.jl]: Oh, ok.

**Pull request merged**